doc/api/http.md

```diff
@@ -542,6 +542,17 @@ changes:
 The `request.aborted` property will be `true` if the request has
 been aborted.
 
+### request.finished
```

Comment:
It seems this section should be placed before `### request.flushHeaders()` ABC-wise.